### DISCUSSION TITLE
No PlantUML on arrange -->insert ---> plantUML for intranet deployment

### DISCUSSION THREAD

**Discussion by jinnjwu** [jgraph/drawio#2045] [Q&A]
No PlantUML on arrange -->insert ---> plantUML for intranet deployment

**Comment by davidjgraph** [comment]
You would need to setup a Plant server and config the instance to use that server

**Reply by jinnjwu** [reply]
yes, started plant server, and configured window.PLANT_URL = 'http://localhost:8080/'; but still not working
